### ISSUE TITLE
Add logic to event Routes

### ISSUE THREAD

**Issue by honjes** [Techeron/fvlr_api#11]
i would like to add returns to the teams and player routs by simple requesting them from the api itself

**Comment by Codokk** [comment]
We could add a URI param (ex refresh=true)

We need to be careful about this though, as it makes it easy to spam requests to vlr.gg

**Comment by honjes** [comment]
Yes, that is true. Maybe we could just check the cache for data already stored and just add that data to the response ? 
That would have the problem of having inconsistent responses, tho.

**Comment by Codokk** [comment]
Maybe we can have some sort of Minimum cache time, (EX: when a record's created timestamp is less than 60 seconds old, they are required to get cached data)

**Comment by honjes** [comment]
Yes, that sounds good. 
maybe we could also add an option to request deeper data (EX: ext =true, when requesting a team also requests the players from vlr). Something like that
